### ISSUE TITLE
Custom menu fields

### ISSUE THREAD

**Issue by anthonytison** [mattmilburn/strapi-plugin-menus#100]
Hi,

Would it be possible to extend the main form (title, slug) ?

I added this customField the same way we can do with the menu items and i can see the field in the database but the form doesn't display the new tag.

```
    const defaultMenuAttrs = strapi.contentType('plugin::menus.menu').attributes;
    strapi.contentType('plugin::menus.menu').attributes = {
      ...defaultMenuAttrs,
      customField: {
        type: 'enumeration',
        enum: [
          'value1', 'value2'
        ]
      }
    }
```

and in the config file i added these lines : 

```

  menus: {
    config: {
      layouts: {
        menu: [
          {
            input: {
              label: 'Custom field',
              name: 'customField',
              type: 'select',
              required: true,
              options: [
                {
                  label: 'label1',
                  value: 'value1'
                },
                {
                  label: 'label2',
                  value: 'value2'
                }
              ]
            }
          }
        ],
     }
  }
```

**Comment by sebsebseb123** [comment]
Hey,
Did you ever get an answer for this? I'm having the same issue. I can see any/all fields that I create in my DB fields, but I don't see anything different in the interface.

**Comment by sebsebseb123** [comment]
Nvmd... looks like it works now. I was using a sandbox using TS and just scraped it all and started over using JS. Works now 😊

**Comment by arioston** [comment]
I don't think we can extend menus with the current configuration, as we can see. [here](https://github.com/mattmilburn/strapi-plugin-menus/blob/87d0cc7280a85987da5d36350c5e0d110b719b33/admin/src/pages/EditView/index.js#L287). It imports the list directly from the file, but for the menuItem, it is merged with the configuration that you fill in.

Maybe we should raise a bug for it? I didn't find any reference in the docs saying that it's possible to extend menus; it only says it's possible to extend MenuItem.

**Comment by mattmilburn** [comment]
Hi @anthonytison @arioston @sebsebseb123 This is not a bug, it's just not implemented yet. A lot of new features will be coming to this plugin before the end of the year if not sooner, including the ability to extend the `Menu` schema. Currently, you can only extend the `MenuItem` schema. Sorry for the wait.

**Comment by prabhjotkaur10** [comment]
Hi @mattmilburn, I also needed the similar functionality to extend the main menu, may I know when is this going to be released. Thanks.
